release-schema.json: add invitationToConfirmInterest, atypicalToolNam… (#3)
open-contracting-extensions/ocds_communication_extension: README.md

# Communication

> Adds a communication object to the tender or a lot to describe the modalities of communication about key events.

Reviewer: ```suggestion
Adds a communication object to the tender and lot objects, to describe the modalities of communication about key events.
```

}
```

### Lot

> An example of a planning notice divided into lots which is also a call for competition.

Reviewer: ```suggestion
An example of a planning notice that is used as a call for competition and that is divided into lots.
```

```json
{
  "tender": {
    "lots": [
      {
        "id": "LOT-0001",
        "communication": {
          "atypicalToolName": "ACertainTool",
          "atypicalToolUrl": "https://ecomm-procurement.example.net",
          "noticeRequestedPublicationDate": "2020-03-15T00:00:00+01:00",
          "documentAvailabilityPeriod": {
            "startDate": "2020-06-15T00:00:00+01:00",
            "endDate": "2020-07-10T00:00:00+01:00"
          },
          "invitationToConfirmInterest": "2020-11-15T09:00:00+01:00"
        }
      }
    ]
  }
}
```

## Issues

Report issues for this extension in the [ocds-extensions repository](https://github.com/open-contracting/ocds-extensions/issues), putting the extension's name in the issue's title.

## Changelog

### 2023-03-09

> * Add `communication.invitationToConfirmInterest`, `communication.atypicalToolName`, `communication.noticeRequestedPublicationDate` and `Lot.communication`

Reviewer: ```suggestion
* Add fields:
  * `Communication.invitationToConfirmInterest`
  * `Communication.atypicalToolName`
  * `Communication.noticeRequestedPublicationDate`
  * `Lot.communication`
```
